plugins/org.jboss.tools.quarkus.cheatsheet/src/org/jboss/tools/quarkus/cheatsheet/OpenCreateProjectWizardAction.java

```diff
@@ -36,14 +36,6 @@ public void run(String[] params, ICheatSheetManager manager) {
 		createProjectWizard.init(workbench, null);
 		WizardDialog dialog = new WizardDialog(window.getShell(), createProjectWizard);
 		dialog.create();
-//		dialog.setPageSize(200, 200);
-//		dialog.getShell().computeSize(200, 200);
-//		dialog.getShell().layout(true);
-//		dialog.getShell().setSize(200,  200);
-//		dialog.getShell().pack();
-		dialog.getShell().pack();
```

Comment:
seems that you removed more than expected...

Reply:
@sbouchet good catch, but this is on purppose. This was a leftover of when I tried to set the wizard shell size. Didn't see that being a "new wizard", this action isn't used when you invoke the wizard via ctrl + n > quarkus

Reply:
ok, thanks for the explanation. I tested again and still looks good on my side.